src/Symfony/Bundle/SecurityBundle/Form/Type/UsernamePasswordType.php

```diff
@@ -42,31 +41,23 @@ public function __construct(RequestStack $requestStack)
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('username', 'Symfony\Component\Form\Extension\Core\Type\TextType')
-            ->add('password', 'Symfony\Component\Form\Extension\Core\Type\PasswordType')
+            ->add('_username', 'Symfony\Component\Form\Extension\Core\Type\TextType')
```

Comment:
Agree.. but if we cant do `EmailType` instead of `TextType` and assign a label or something im not sure how convenient this actually is. Ie. i think i prefer having this definition in the application layer anyway (like tests did before).